android/src/main/java/com/usabilla/reactnative/ubform/UsabillaBridge.java

```diff
@@ -166,6 +166,18 @@ public void setDefaultNavigationButtonsVisibility(@NonNull final boolean visible
         usabilla.setDefaultNavigationButtonsVisibility(visible);
     }
 
+    /**
+     * Called via the index.js to download passive forms and cache them
+     */
+    @ReactMethod
+    public void preloadFeedbackForms(@NonNull final ReadableArray formIDs) {
```

Comment:
Out of curiosity, we use readable arrays to pass data from JS to java, right?

Reply:
yes @demirsoya